sympy/utilities/lambdify.py

```diff
@@ -151,10 +149,20 @@ def _import(module, reload="False"):
     for sympyname, translation in translations.items():
         namespace[sympyname] = namespace[translation]
 
+    # For computing the modulus of a sympy expression we use the builtin abs
+    # function, instead of the previously used fabs function for all
+    # translation modules. This is because the fabs function in the math
+    # module does not accept complex valued arguments. (see issue 9474). The
+    # only exception, where we don't use the builtin abs function is the
+    # mpmath translation module, because mpmath.fabs returns mpf objects in
+    # contrast to abs().
```

Comment:
If we want that ``lambdify(..., modules='mpmath')`` always returns mpf objects, using ``mpmath.fabs`` in the translation makes sense. But what should be returned by lambdify if the ``modules`` kwarg is ``'['mpmath', 'numpy']``?  BTW ``lambdify(x, x+S(1)/2, modules='mpmath')(0)`` currently does not return a mpf object.